<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="windowModality">
   <enum>Qt::WindowModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>956</width>
    <height>761</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QComboBox" name="comboGraf">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>70</y>
      <width>211</width>
      <height>29</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>Puntos</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Barras</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Torta</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Linea</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Regresi칩n lineal</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Reg cuadr치tica</string>
     </property>
    </item>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>40</y>
      <width>201</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Tipo de gr치fico</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>120</y>
      <width>201</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Vble independiente</string>
    </property>
   </widget>
   <widget class="QComboBox" name="comboX">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>150</y>
      <width>211</width>
      <height>29</height>
     </rect>
    </property>
   </widget>
   <widget class="QComboBox" name="Serie">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>220</y>
      <width>211</width>
      <height>29</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>190</y>
      <width>201</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Serie/s</string>
    </property>
   </widget>
   <widget class="QComboBox" name="Serie_2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>260</y>
      <width>211</width>
      <height>29</height>
     </rect>
    </property>
   </widget>
   <widget class="QComboBox" name="Serie_3">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>300</y>
      <width>211</width>
      <height>29</height>
     </rect>
    </property>
   </widget>
   <widget class="QComboBox" name="Serie_4">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>340</y>
      <width>211</width>
      <height>29</height>
     </rect>
    </property>
   </widget>
   <widget class="otraClase" name="otraClase">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>0</y>
      <width>691</width>
      <height>701</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>10</y>
      <width>201</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="graficar">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>620</y>
      <width>201</width>
      <height>61</height>
     </rect>
    </property>
    <property name="text">
     <string>Graficar</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>956</width>
     <height>26</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuArchivo">
    <property name="title">
     <string>Archivo</string>
    </property>
    <addaction name="actionCerrar"/>
    <addaction name="actionGuardar_gr_fico"/>
    <addaction name="actionCargar_datos"/>
   </widget>
   <addaction name="menuArchivo"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionCerrar">
   <property name="text">
    <string>Cerrar</string>
   </property>
  </action>
  <action name="actionGuardar_gr_fico">
   <property name="text">
    <string>Guardar gr치fico</string>
   </property>
  </action>
  <action name="actionCargar">
   <property name="text">
    <string>Cargar</string>
   </property>
  </action>
  <action name="actionCargar_datos">
   <property name="text">
    <string>Cargar datos</string>
   </property>
  </action>
 </widget>
 <customwidgets>
  <customwidget>
   <class>otraClase</class>
   <extends>QGraphicsView</extends>
   <header>otraClase.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
